bynder_local.module

```diff
@@ -147,7 +147,7 @@ function bynder_local_get_image_url(array $item) {
     }
   }
   else {
-    return $item['thumbnails'][$config->get('derivative')] ?? NULL;
+    return $item['thumbnails'][$config->get('derivative')] ?? $item['thumbnails']['webimage'];
```

Comment:
Could you add a comment here to describe why we're doing this, and add a todo to provide an indication that we need to replace this with the link to the original when available?

Reply:
Done.